### ISSUE TITLE
Follow up work on MSHV related changes into CLH

### ISSUE THREAD

**Issue by russell-islam** [cloud-hypervisor/cloud-hypervisor#2048]
Here are the list of work that were promised to do in later PRs

1.  Remove Clippy suppression from mshv/mod.rs
2. Move macro_rules! msr { } to hypervisor/src/arch/x86

**Comment by liuw** [comment]
Is this a good place track more MSHV related work items? I have some in mind too:

1. dev_cli.sh support for MSHV.
1. The x86 instruction emulator should be gated by MSHV. KVM does not need it. This depends on the previous item.

**Comment by russell-islam** [comment]
Here are two big items I have created to track:
https://github.com/cloud-hypervisor/cloud-hypervisor/issues/2049
https://github.com/cloud-hypervisor/cloud-hypervisor/issues/2050

**Comment by rbradford** [comment]
> Here are the list of work that were promised to do in later PRs
> 
>     1. Remove Clippy suppression from mshv/mod.rs
>     2. Move macro_rules! msr { } to hypervisor/src/arch/x86

Any update on these?

**Comment by liuw** [comment]
Item 2 is already done. I will have a look at item 1. At first glance it should be easily fixable at the stage.
